tests/tester/helpers.go

```diff
@@ -46,22 +47,27 @@ func (t Tester) MakeTestBundle(name string, ref string) {
 	t.RequirePorter("publish", "--reference", ref)
 }
 
-func (t Tester) ShowInstallation(namespace string, name string) (claims.Installation, error) {
+func (t Tester) ShowInstallation(namespace string, name string) (porter.DisplayInstallation, error) {
 	stdout, _, err := t.RunPorter("show", name, "--namespace", namespace, "--output=json")
 	if err != nil {
-		return claims.Installation{}, err
+		return porter.DisplayInstallation{}, err
 	}
 
-	var installation claims.Installation
-	require.NoError(t.T, json.Unmarshal([]byte(stdout), &installation))
-	return installation, nil
+	var di porter.DisplayInstallation
+
+	require.NoError(t.T, json.Unmarshal([]byte(stdout), &di))
+
+	return di, nil
 }
 
 func (t Tester) RequireInstallationExists(namespace string, name string) claims.Installation {
-	installation, err := t.ShowInstallation(namespace, name)
+	di, err := t.ShowInstallation(namespace, name)
 	require.NoError(t.T, err)
-	require.Equal(t.T, name, installation.Name, "incorrect installation name")
-	require.Equal(t.T, namespace, installation.Namespace, "incorrect installation namespace")
+	require.Equal(t.T, name, di.Name, "incorrect installation name")
+	require.Equal(t.T, namespace, di.Namespace, "incorrect installation namespace")
+	installation, err := di.ConvertToInstallation()
```

Comment:
This function `RequireInstallationExists` is meant to check that we persisted the installation, so I'd prefer that we don't add checks for specific features here.

Instead of asserting this functionality across all the smoke tests, let's add a test case to one of the existing smoke tests to verify that sensitive data was persisted correctly. At the smoke test level I'm much more concerned about if we persisted a secret to the secret store, and that the installation renders properly.

This check here probably belongs in a unit test in the same file where ConvertToInstallation is defined with accompanying asserts that it was converted properly.

Reply:
This change is required here is due to the fact that now `ShowInstallation()` returns the type of `DisplayInstallation` rather than `claims.Installation`. However, it seems that this function expects a claim record to be returned. Maybe that's not necessary, I will double check

Reply:
I haven't looked in the code, but I think it's safe to return the display installation here instead of claims.Installation